package.json

```diff
@@ -49,7 +49,7 @@
   "peerDependencies": {
     "@fastify/swagger": ">=9.5.1",
     "fastify": "^5.0.0",
-    "zod": ">=3.25.67"
+    "zod": ">=3.25.67 || ^4.0.0"
```

Comment:
isn't >=3 already including 4?

Reply:
Sorry, that's absolutely true 😅.

How about change it something like:

```suggestion
    "zod": "^3.25.67 || ^4.0.0"
```

This would avoid that we declare that we support a possible future zod 5 version.

If not feel free to close the PR.

Reply:
sure, that works